source/base/Debug.ooc

```diff
@@ -34,8 +34,14 @@ Debug: class {
 		This printFunction = f
 	}
 	print: static func (printOut: String, level: Int = 1) {
-		if (This _level == level || (This _level == DebugLevel Everything) ) {
-			This printFunction(printOut)
+		version(debugPrint) {
+			if (This _level == level || (This _level == DebugLevel Everything) ) {
```

Comment:
What if `level < This _level`? Pretty sure the idea is that if the level of a message is above or equal to the current level, it prints, otherwise it doesn't. So if the level is "notification", you print everything. This requires the DebugLevel enum to be sorted in order of severity, of course...